deployment/k8s-config/kustomize/overlays/keel-dev/skaha/skaha-tomcat-deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
         - name: skaha.sessionexpiry
           value: "345600"
-        image: images-rc.canfar.net/skaha-system/skaha:0.10.0
+        image: images-rc.canfar.net/skaha-system/skaha:0.10.1
```

Comment:
Current released version is 0.11.0, so increment from there.  See `VERSION` file.

Reply:
Yes, pulled from master and updated to 0.11.1.